asllib/ASTUtils.mli

```diff
@@ -336,5 +336,8 @@ val list_fold_left_map :
 (** [fold_left_map] is a combination of [fold_left] and [map] that threads an
     accumulator through calls to [f]. Taken from stdlib 4.11. *)
 
+val list_nub : 'a list -> 'a list
```

Comment:
How about `remove_duplicates` or `uniq` (similar to the standard `sort_uniq`)?

Reply:
I like `uniq`! Will update